applications/tari_base_node/src/builder.rs

```diff
@@ -158,7 +159,9 @@ pub fn configure_and_initialize_node(
         DatabaseType::Memory => {
             let backend = MemoryDatabase::<HashDigest>::default();
             let db = BlockchainDatabase::new(backend).map_err(|e| e.to_string())?;
-            let (comms, handles) = setup_comms_services(&rt, id.clone(), peers, &config.peer_db_path, db.clone());
+            let mempool = Mempool::new(db.clone(), MempoolConfig::default());
```

Comment:
Can you make an issue to put Mempool config vars in the config file?

Reply:
I created an issue to update the config.